packages/medusa/src/api/routes/admin/store/update-store.js

```diff
@@ -4,6 +4,7 @@ export default async (req, res) => {
   const schema = Validator.object().keys({
     name: Validator.string(),
     swap_link_template: Validator.string(),
+    payment_link_template: Validator.string(),
```

Comment:
Should we have a separate link for draft orders - now that we have a separate cart type for draft orders.